manageiq-providers-amazon.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,lib}/**/*"]
 
-  s.add_dependency("aws-sdk", ["~>2.9.7"])
+  s.add_dependency("aws-sdk", ["~> 3"])
```

Comment:
Same comment regarding the gem version, and also remove all of the parens and square brackets (go for consistency with the lines below)

Reply:
done